app/controllers/adoption_applications_controller.rb

```diff
@@ -1,23 +1,33 @@
 class AdoptionApplicationsController < ApplicationController
   def index
-    @adoption_applications = AdoptionApplication.all
   end
 
   def show
     @adoption_application = AdoptionApplication.find(params[:id])
+    @pet = Pet.find_by(name: params[:name])
+    @pet = params[:name] if params[:name] && @pet.nil?
```

Comment:
I'm not fully understanding the second @pet variable you're exposing here - let's review when we meet next.